tests/integration/test_custom_comparator.py

```diff
@@ -48,29 +48,27 @@ def generate_snapshots(testdir, testcases_initial):
     return result, testdir, testcases_initial
 
 
-@pytest.mark.xfail(strict=False)
```

Comment:
The xfail here is because the test is flakey, haven't figured out why.

Reply:
I've put the flaky markers back in.
Until my PR is merged, the workflow has to be triggered manually :( but hopefully once my first PR is in I can tweak more easily...